Sources/SwerverCore/Server/ClientRequestHandler.swift

```diff
@@ -0,0 +1,28 @@
+import Foundation
+import Socket
+
+public class ClientRequestHandler {
```

Comment:
`ClientRequestHandler` is not a great name. Based on our conversation regarding the design of these app, can you think of a better name? To give you a hint, it _was_ a name we discussed using. :)

Another way of looking at it: What is the responsibility of this class?